child_compassion/models/compassion_hold.py

```diff
@@ -64,6 +66,17 @@ def release_hold(self):
             self.child_id.active = False
             message_obj.create(message_vals)
 
+    @api.model
+    def check_hold_validity(self):
+        expired_holds = self.env['compassion.hold'].search([
+            ('expiration_date', '<',
+             datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
+        ])
+
+        for hold in expired_holds:
+            hold.active = False

```

Comment:
this will trigger n queries in the database. If you put `expired_holds.write({'active': False})` instead, it will only trigger one query.